proto/beacon/rpc/v1/services.proto

```diff
@@ -53,18 +58,42 @@ message AttestResponse {
     bytes attestation_hash = 1;
 }
 
+// Request assignment updates for either all validators or a subset of validators 
+// defined by their public keys.
+message ValidatorAssignmentRequest {
+    bool all_validators = 1;
+    repeated PublicKey public_keys = 2;
+}
+
+message ValidatorAssignmentResponse {
```

Comment:
Does a validator need to know anything more than this?

- Slot number for assignment
- Role: Propose / validate 
- Shard ID

I think that captures everything. What does everyone else think?